proto/v1/database_service.proto

```diff
@@ -295,6 +304,21 @@ message BatchUpdateDatabasesResponse {
   repeated Database databases = 1;
 }
 
+message BatchSyncDatabasesRequest {
+  // The parent resource shared by all databases being updated.
+  // Format: instances/{instance}
+  // If the operation spans parents, a dash (-) may be accepted as a wildcard.
+  string parent = 1;
+
+  // The list of database names to retrieve.
+  repeated string names = 2 [
```

Comment:
I'm not sure whether this should be `repeated string names` or `repeated SyncDatabaseRequest requests`. aip doesn't cover custom batch methods.
https://google.aip.dev/234

Reply:
I think `repeated string names` should be fine as long as we are not going to add new fields to `SyncDatabaseRequest`.